incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionFactory.java

```diff
@@ -2842,7 +3048,24 @@ private void onKafkaBegin(
 
             assert replyAck <= replySeq;
 
-            delegate.doMqttBegin(traceId, authorization, affinity);
+            Flyweight mqttBeginEx = EMPTY_OCTETS;
+
+            //TODO: decide where to do this? How will the group stream reject the invalid timeout?
```

Comment:
The group stream will not reject the invalid timeout, instead it will send back a `KafkaGroupBeginEx` with a timeout that works, ideally the same as the one requested, but possibly updated to fall with the supported range of kafka consumer group session timeout values.

Reply:
Nice, by reject I meant restrict, but it was the wrong word phrase for sure. It'll be an easy change once it's ready